qa/qa_utils.py

```diff
@@ -347,7 +347,7 @@ def StartLocalCommand(cmd, _nolog_opts=False, log_cmd=True, **kwargs):
       pcmd = cmd
     print("%s %s" % (colors.colorize("Command:", colors.CYAN),
                      utils.ShellQuoteArgs(pcmd)))
-  return subprocess.Popen(cmd, shell=False, **kwargs)
+  return subprocess.Popen(cmd, shell=False, text="utf-8", **kwargs)
```

Comment:
This and below confuse me greatly. The documentatio for Popen() argument states that the `text` argument:

- is an alias for universal_newlines, which is a boolean
- and was added in 3.7 only (we're still targetting 3.5 or 3.6?)

Did you mean instead `encoding` here?

Reply:
Yes, I meant encoding, but misread the documentation. This worked because "utf-8" evaluates to True, but I'll fix it.